corehq/apps/export/views.py

```diff
@@ -872,6 +872,14 @@ def get_filters(self, filter_form_data):
         filter_form = self._get_filter_form(filter_form_data)
         return filter_form.get_case_filter()
 
+    def _get_filter_form(self, filter_form_data):
+        filter_form = self.filter_form_class(
+            self.domain_object, filter_form_data
```

Comment:
Seems urgent though happy to merge since clearly no tests

On Thursday, February 25, 2016, Ben Rudolph <brudolph@dimagi.com> wrote:

> Tricky - +1 after tests
>
> On Thursday, February 25, 2016, Noah Carnahan <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> In corehq/apps/export/views.py
>> <https://github.com/dimagi/commcare-hq/pull/10489#discussion_r54096236>:
>>
>> > @@ -872,6 +872,14 @@ def get_filters(self, filter_form_data):
>> >          filter_form = self._get_filter_form(filter_form_data)
>> >          return filter_form.get_case_filter()
>> >
>> > +    def _get_filter_form(self, filter_form_data):
>> > +        filter_form = self.filter_form_class(
>> > +            self.domain_object, filter_form_data
>>
>> There's no timezone arg here
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/dimagi/commcare-hq/pull/10489/files#r54096236>.
>>
>